packages/cli/commands/upload.js

```diff
@@ -204,45 +253,30 @@ exports.handler = async options => {
       absoluteSrcPath,
       options.convertFields
     );
-    uploadFolder(
+
+    const noModules = filePaths.filter(path => !path.includes('/modules/'));
```

Comment:
We have to be careful when checking paths. Using slashes like this may result in bugs for certain filesystems that use backslash (like windows). 

You may be able to use some of the utils that we have like this one: https://github.com/HubSpot/hubspot-cli/blob/master/packages/cli-lib/modules.js#L47